tasks/svg-sprite.js

```diff
@@ -32,52 +33,57 @@ const config = {
 };
 
 function cleanIcons() {
-  return del(`${svgPath}/usa-icons`);
+  return del([`${svgRootPath}/usa-icons`, `${svgPackagePath}/usa-icons`]);
 }
 
 function collectIcons() {
   dutil.logMessage("collectIcons", "Collecting default icon set in dist/img/usa-icons");
   return src([
-    `node_modules/@material-design-icons/svg/outlined/{${iconConfig.material}}.svg`,
+    `node_modules/@material-design-icons/svg/filled/{${iconConfig.material}}.svg`,
     `packages/usa-icon/src/img/material-icons-deprecated/{${iconConfig.materialDeprecated}}.svg`,
     `packages/usa-icon/src/img/uswds-icons/{${iconConfig.uswds}}.svg`,
   ])
-    .pipe(dest(`${svgPath}/usa-icons`))
+    .pipe(dest(`${svgPackagePath}/usa-icons`));
 }
 
 function buildSprite(done) {
   return (
-    src(`${svgPath}/usa-icons/*.svg`)
+    src(`${svgPackagePath}/usa-icons/*.svg`)
       .pipe(svgSprite(config))
       .on("error", logError)
-      .pipe(dest(svgPath))
+      .pipe(dest(svgPackagePath))
       .on("end", () => done())
   );
 }
 
 function renameSprite() {
-  return src(`${svgPath}/symbol/svg/sprite.symbol.svg`)
-    .pipe(rename(`${svgPath}/sprite.svg`))
+  return src(`${svgPackagePath}/symbol/svg/sprite.symbol.svg`)
+    .pipe(rename(`${svgPackagePath}/sprite.svg`))
     .pipe(dest(`./`));
 }
 
 function cleanSprite() {
-  return del(`${svgPath}/symbol`);
+  return del([`${svgPackagePath}/symbol`]);
 }
 
+// create sprite and gather all usa-icon assets in package dist
 exports.buildSpriteStandalone = series(
-  copyIcons,
+  copyMaterialIcons,
+  copyPackageIcons,
   cleanIcons,
   collectIcons,
   buildSprite,
   renameSprite,
-  cleanSprite
+  cleanSprite,
+  copyIconAssets
```

Comment:
As the last task of the buildSprite tasks, copy the img assets inside usa-icon to the root dist directory. 

This line can be removed if/when we stop distributing icons to root dist/img.

Reply:
Moving usa-icon scripts to that directory would increase clarity.

Reply:
Agreed. Moved the package-specific scripts in [180b36b](https://github.com/uswds/uswds/pull/4878/commits/180b36b237b9ea8e36b42c4f3b8e05bad341759d)